LayoutTests/svg/markers/marker-mixing-repaint-rect-and-bounding-box-expected.html

```diff
@@ -0,0 +1,22 @@
+<style>
+  .line {
+    position: absolute;
+    display: block;
+    width: 100%;
+    height: 100%;
+    overflow: visible;
+    min-height: 1px;
+    min-width: 1px;
+    stroke: red;
+    fill: red;
+  }
+</style>
```

Comment:
Some of these properties are not needed to reproduce the bug. So you could reduce this further if you want.

Reply:
Done!